<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Triad: Triangle Mesh Refinment</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenMesh" />
    <addaction name="actionSaveMesh" />
    <addaction name="separator" />
    <addaction name="actionExportImage" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuSelect" >
    <property name="title" >
     <string>&amp;Select</string>
    </property>
    <addaction name="actionOnlyOne" />
    <addaction name="actionManualSelection" />
    <addaction name="separator" />
    <addaction name="actionByLongestEdge" />
    <addaction name="actionByTriangleArea" />
    <addaction name="separator" />
    <addaction name="actionClearSelection" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuSelect" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionOpenMesh" >
   <property name="text" >
    <string>&amp;Open Mesh</string>
   </property>
   <property name="toolTip" >
    <string>Opens a saved mesh</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveMesh" >
   <property name="text" >
    <string>&amp;Save Mesh</string>
   </property>
   <property name="toolTip" >
    <string>Save a refined mesh</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExportImage" >
   <property name="text" >
    <string>Export &amp;Image</string>
   </property>
   <property name="toolTip" >
    <string>Export image</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOnlyOne" >
   <property name="text" >
    <string>&amp;Only One</string>
   </property>
  </action>
  <action name="actionByLongestEdge" >
   <property name="text" >
    <string>By &amp;Longest Edge</string>
   </property>
  </action>
  <action name="actionByTriangleArea" >
   <property name="text" >
    <string>By &amp;Triangle Area</string>
   </property>
  </action>
  <action name="actionClearSelection" >
   <property name="text" >
    <string>&amp;Clear Selection</string>
   </property>
  </action>
  <action name="actionManualSelection" >
   <property name="text" >
    <string>&amp;Manual Selection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
